python/paddle/nn/functional/pooling.py

```diff
@@ -2343,3 +2343,249 @@ def fractional_max_pool3d(
         )
 
         return (pool_out, mask) if return_mask else pool_out
+
+
+def lp_pool1d(
+    x,
+    norm_type,
+    kernel_size,
+    stride=None,
+    padding=0,
+    ceil_mode=False,
```

Comment:
Check `lp_pool2d` has the data_format parameter. Do we also need it in `lp_pool2d` here